test-clients/src/main/java/com/hedera/services/bdd/suites/token/UniqueTokenManagementSpecs.java

```diff
@@ -121,28 +141,41 @@ private HapiApiSpec happyPathOneMintFiveMetadata() {
 								ByteString.copyFromUtf8("memo4")
 						))
 				).then(
-						getTokenNftInfo("token", 1).hasSerialNum(1).hasMetadata(ByteString.copyFromUtf8("memo")),
-						getTokenNftInfo("token", 2).hasSerialNum(2).hasMetadata(ByteString.copyFromUtf8("memo1")),
-						getTokenNftInfo("token", 3).hasSerialNum(3).hasMetadata(ByteString.copyFromUtf8("memo2")),
-						getTokenNftInfo("token", 4).hasSerialNum(4).hasMetadata(ByteString.copyFromUtf8("memo3")),
+						getTokenNftInfo(TOKEN_NAME, 1)
+								.hasSerialNum(1)
+								.hasMetadata(ByteString.copyFromUtf8("memo")),
+						getTokenNftInfo(TOKEN_NAME, 2)
+								.hasSerialNum(2)
+								.hasMetadata(ByteString.copyFromUtf8("memo1")),
+						getTokenNftInfo(TOKEN_NAME, 3)
+								.hasSerialNum(3)
+								.hasMetadata(ByteString.copyFromUtf8("memo2")),
+						getTokenNftInfo(TOKEN_NAME, 4)
+								.hasSerialNum(4)
+								.hasMetadata(ByteString.copyFromUtf8("memo3")),
 
-						getTokenNftInfo("token", 5).hasSerialNum(5)
+						getTokenNftInfo(TOKEN_NAME, 5).hasSerialNum(5)
 								.hasMetadata(ByteString.copyFromUtf8("memo4"))
 								.hasTokenID(TOKEN_NAME)
-								.hasAccountID(TOKEN_TREASURY)
+								.hasAccountID(TOKEN_TREASURY),
+
+						getAccountInfo(TOKEN_TREASURY)
+								.hasToken(relationshipWith(TOKEN_NAME))
 				);
 	}
 
-	private HapiApiSpec happyPathFiveMintOneMetadata() {
-		return defaultHapiSpec("happyPathFiveMintOneMetadata")
+	private HapiApiSpec happyPathMintMultipleWithIdenticalMetadata() {
```

Comment:
Question: I though that we have a check - if there are identical memo's we are failing with error response code. From this test it seems that this is not the case. Lets ignore this use-case since we have to discuss what happens when we have `empty metadata bytes` or `identical metadata bytes`

Reply:
leaving it unresolved for now

Reply:
We actually had a check, but it's not applicable in this test, as there are 2 actual batches here, and each batch contains a single-element list with the same data inside.